pulp_python/tests/functional/api/test_crud_publications.py

```diff
@@ -25,19 +20,19 @@
     PYTHON_EGG_FILENAME,
     PYTHON_WHEEL_FILENAME,
 )
-from pulp_python.tests.functional.utils import gen_python_client, gen_python_remote
+from pulp_python.tests.functional.utils import (
+    create_workflow,
+    ensure_simple,
+    gen_repo_sync_publish_workflow,
+    gen_repo_sync_publish_distribute_workflow,
+    WorkflowStep as WS,
+    repo_api,
+    pub_api,
+    distro_api,
```

Comment:
These feel a bit over-shortened, and I think since most other plugins use `publication_api`, `distribution_api` and so forth, we should probably continue the pattern.  An outside developer should be able to look at the code and understand what's going on without knowing the shorthands.